tools/clang/test/HLSLFileCheck/hlsl/diagnostics/errors/write_const_arrays.hlsl

```diff
@@ -0,0 +1,61 @@
+// RUN: %dxc -T cs_6_0 _HV 2021 %s | FileCheck %s
```

Comment:
Can we use verifier tests for front-end error checks?

Reply:
I suppose so.